2. ročník/jazyky/C/Vypocet Uctu/VypocetUctu.c

```diff
@@ -1,12 +1,26 @@
+/*
+ * VypocetUctu.c ver. 1.1
+ *
+ * Program na vypocet uctu s urokem 
+ * za cely rok a vkladem kazdy mesic
+ * =================================
+ *
+ * v1.0 - Elias Constantine - 25.9.2024 
+ * v1.1 - František Rubáček - 9.10.2024 - upravil jsem köd pro výpis do souboru pro lepší čitelnost, spravený uživatelský vstup
+ * 
+ */
+
 #include <stdio.h>
-#define UROKOVAMIRA 5.61
+#include <stdlib.h> //pro exit()
+
+#define UROKOVAMIRA 5.61 
 #define MINVKLAD 100
 #define MAXVKLAD 100000
-#define DAN 15
+#define DAN 15 // <- nemám tucha co je toto, vypadá to na den po kterém chodí výplata, jelikož každý měsíc 16. je hodnota výrazně odlišná od jiné. Musí to okomentovat Eliáš.
```

Comment:
@Eliasko6562 prosimtě, napíšeš k tomu co přesně to je?

Reply:
ok